Hide home ads section

system32uwu opened the pull request:
In case someone is not using an adblock plugin / module for whatever reason they will get this

Daksh777 commented:
I've reverted the reverted PR again because it doesn't affect #4, at least on my PC.

system32uwu commented:
Interesting, what's your Spotify version? I just switched to arch yesterday and installed the latest spotify package from AUR and added this in my user.css, I got that error message but closing and opening Spotify again made the error disappear.

Daksh777 commented:
You're right, this doesn't affect Spotify's home page. The user must be having some other problem.
